search/lib/convert/index.js

```diff
@@ -1,7 +1,9 @@
 const boundingBox = require('./bounding-box');
 const collections = require('./collections');
+const granuleToItem = require('./granule-to-item');
```

Comment:
I think the filename, a.k.a. the module, could just be called Granules or Items.

Reply:
Probably Granules since that is what this module is converting from.